web/tls_config.go

```diff
@@ -51,6 +51,7 @@ type TLSConfig struct {
 	MinVersion               TLSVersion `yaml:"min_version"`
 	MaxVersion               TLSVersion `yaml:"max_version"`
 	PreferServerCipherSuites bool       `yaml:"prefer_server_cipher_suites"`
+	ClientCertAllowedSanDNS  string     `yaml:"client_cert_allowed_san_dns"`
```

Comment:
Should we future proof this a bit and make it a list of allowed SANs? Or maybe an allow regex?

Reply:
good idea, ended up going with an allow regex that checks against all SAN fields